sqlglot/dialects/sqlite.py

```diff
@@ -116,6 +118,7 @@ class Parser(parser.Parser):
             "STRFTIME": _build_strftime,
             "DATETIME": lambda args: exp.Anonymous(this="DATETIME", expressions=args),
             "TIME": lambda args: exp.Anonymous(this="TIME", expressions=args),
+            "INSTR": lambda args: exp.StrPosition(substr=seq_get(args, 1), this=seq_get(args, 0)),
```

Comment:
Move this to the base `Parser`'s `FUNCTIONS` dict and also remove MySQL's entry as well.